src/packet.proto

```diff
@@ -2,6 +2,23 @@ syntax = "proto3";
 
 package helium;
 
+message gps_info {
+    uint32 horizontal_accuracy = 1; // meters? this will take into account h3 precision and distance from asserted location
+    uint32 vertical_accuracy = 2; // meters?
+    int32 altitude = 3; // meters?
+    uint32 time_accuracy = 4; //nanoseconds
+}
+
+// https://www.eecis.udel.edu/~mills/ntp/html/ntpq.html#system
+message ntp_info {
+    bool synced = 1;
+    uint32 stratum = 2;
+    float sys_jitter = 3; //combined system jitter, in milliseconds
+    float clock_jitter = 4; //clock jitter, in milliseconds
```

Comment:
Do these have to be floats? vs some other fixed point representation?

Reply:
ntp supplies them as floats